pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.ge.predix</groupId>
     <artifactId>spring-metering-filter</artifactId>
-    <version>1.2.2-SNAPSHOT</version>
+    <version>1.2.3</version>
```

Comment:
Reminder to change this to 1.2.3-SNAPSHOT once UAA has verified that their metering integration test passes